docs/content/stable/yugabyte-platform/back-up-restore-universes/back-up-universe-data.md

```diff
@@ -122,7 +123,8 @@ For example:
 ```output
 s3://user_bucket
   /some/sub/folders
-    /univ-a85b5b01-6e0b-4a24-b088-478dafff94e4
+    /univ-univ_name-a85b5b01-6e0b-4a24-b088-478dafff94e4
```

Comment:
shouldn't it be
univ-univ-name_a85b5b01-6e0b-4a24-b088-478dafff94e4
or even
universe-name_a85b5b01-6e0b-4a24-b088-478dafff94e4